src/sentry/api/bases/avatar.py

```diff
@@ -25,13 +25,29 @@ def validate(self, attrs):
         return attrs
 
 
+class SentryAppLogoSerializer(serializers.Serializer):
+    avatar_photo = SentryAppLogoField(required=True)
+    avatar_type = serializers.ChoiceField(choices=(("upload", "upload")))
+    color = serializers.BooleanField(required=False)
+
+    def validate(self, attrs):
+        attrs = super().validate(attrs)
+        if attrs.get("avatar_type") != "upload":
+            raise serializers.ValidationError(
+                {"avatar_type": "Cannot set avatar_type to anything but upload."}
```

Comment:
nit: I think the error would be clearer as "avatar_type must be set to 'upload'"